pkg/bff/admin/client.go

```diff
@@ -29,7 +29,7 @@ func New(conf config.AdminConfig) (client admin.DirectoryAdministrationClient, e
 		return nil, err
 	}
 
-	return admin.New(conf.Endpoint, creds)
+	return admin.New(conf.Endpoint, creds, nil)
```

Comment:
Is the `nil` argument still required here?

Reply:
Thank you for catching this, I knew I probably missed one!